lib/logstash/outputs/s3.rb

```diff
@@ -288,14 +296,6 @@ def aws_service_endpoint(region)
     { :s3_endpoint => region == 'us-east-1' ? 's3.amazonaws.com' : "s3-#{region}.amazonaws.com"}
   end
 
-  def upload_options
```

Comment:
I've made the method public to unit test it. :(
There was a layout issue in the changelog so I have corrected it.